plugins/ssr/ssrparser.ml4

```diff
@@ -342,7 +342,7 @@ let interp_index ist gl idx =
 
 open Pltac
 
-ARGUMENT EXTEND ssrindex TYPED AS ssrindex PRINTED BY pr_ssrindex
+ARGUMENT EXTEND ssrindex PRINTED BY pr_ssrindex
```

Comment:
Why is this needed? I did not notice this before. The semantics is not the same if ssrindex has non trivial functions attached to it.

Reply:
This is needed as to remove a warning "ignored TYPED AS" annotation.

Reply:
Maybe the right thing to do is to remove the `mli` file? I dunno, in 8.10 certainly things are different now.

Reply:
ok then.